tests/unit/utils/converter_unittest.cc

```diff
@@ -0,0 +1,84 @@
+#include "sg_gtest_utilities.h"
+
+#include <gtest/gtest.h>
```

Comment:
move this to be the first included header plz

Reply:
Moved.